pkg/arvo/app/permission-group-hook.hoon

```diff
@@ -0,0 +1,179 @@
+/-  *group-store, *permission-store, *permission-group-hook
```

Comment:
A brief comment describing what exactly is an association, I think I had the direction backwards in my mind.  To be clear, this lets you say "any time I add a person to this group also add them to this set of permissions groups".  For example, you could have five chat rooms all whitelisted to one particular group, and when you add someone to that group they'll have permission to join all five rooms.  Is that correct?

Reply:
Yes, that’s correct.